src/commands/dev-env-sync-sql.js

```diff
@@ -174,8 +174,8 @@ export class DevEnvSyncSQLCommand {
 			await unzipFile( this.gzFile, this.sqlFile );
 			console.log( `${ chalk.green( '✓' ) } Extracted to ${ this.sqlFile }` );
 		} catch ( err ) {
-			await this.track( 'archive_extraction_error', { errorMessage: err?.message } );
```

Comment:
Why this change? Is there a lint warning or something? We had a bug in upgrade wp site recently where library promised to never return null via types and it did.

Reply:
Because `err` will be an instance of `Error`, it cannot be `null` or `undefined`. See https://github.com/Automattic/vip-cli/pull/1391/files#diff-3e887d690f0e63a71d15440119be0462cb0d8b436c200a7425ea9f891d395dc7R106-R109

If `pipeline()` fails, it will reject with `Error`.

Reply:
> where library promised to never return null via types and it did

It is likely that type definitions for the library were compiled with [`strictNullChecks: false`](https://www.typescriptlang.org/tsconfig#strictNullChecks).